pkg/pipelines/add_service_test.go

```diff
@@ -0,0 +1,36 @@
+package pipelines
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/openshift/odo/pkg/pipelines/ioutils"
+	"github.com/spf13/afero"
+)
+
+func TestService(t *testing.T) {
+
+	fakeFs := ioutils.NewFilesystem()
+	gitopsPath := afero.GetTempDir(fakeFs, "test")
+	manifestFile := "config/testdata/pipelines.yaml"
```

Comment:
Here the manifest file is referred from the underlying filesystem. It should be added to the in-memory filesystem (fakeFs)